Dockerfile

```diff
@@ -21,53 +21,62 @@ RUN --mount=type=cache,target=/go/pkg/mod/ \
 # source code into the container.
 RUN --mount=type=cache,target=/go/pkg/mod/ \
     --mount=type=bind,target=. \
-<<EOF
+    <<EOF
     export VERSION_PKG_PATH="go.autokitteh.dev/autokitteh/internal/version"
     export TIMESTAMP="$(date -u "+%Y-%m-%dT%H:%MZ")"
-    export LDFLAGS="-X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
+    export LDFLAGS="-s -w -X "${VERSION_PKG_PATH}.Version=$(cat .version || echo)" -X "${VERSION_PKG_PATH}.Time=${TIMESTAMP}" -X "${VERSION_PKG_PATH}.Commit=$(cat .commit || echo)""
```

Comment:
On my machine this reduces `ak` size from 158MB to 114MB. Do we care that much about 44MB? Does it worth losing debug information over it?

Reply:
lower image size allows us to spin more instances, loads them faster to memory, cost reduction on moving bytes in AWS and so on...

we can have a debug and final image if we want